contracts/schemes/bootstrap/DxGenAuction4Rep.sol

```diff
@@ -5,6 +5,6 @@ import "@daostack/arc/contracts/schemes/Auction4Reputation.sol";
 /**
  * @title Scheme for conducting ERC20 Tokens auctions for reputation
  */
-contract DxAuction4Rep is Auction4Reputation {
+contract DxGenAuction4Rep is Auction4Reputation {
     constructor () public {}
 }
```

Comment:
does this CR is needed ?

Reply:
Is not "required" it's just more cleat what the scheme is for now.